at_client/lib/src/manager/at_client_manager.dart

```diff
@@ -30,24 +30,27 @@ class AtClientManager {
 
   AtClientManager._internal();
 
+  @deprecated
```

Comment:
Shouldn't deprecate, as this is still a valid usage by apps. We're just deprecating for internal usage within the library for structural and testability reasons.